recovery/translation_it.ts

```diff
@@ -101,22 +101,22 @@
         <location filename="bootselectiondialog.ui" line="45"/>
         <source>&amp;Up</source>
         <oldsource>Up</oldsource>
-        <translation type="unfinished">Su</translation>
+        <translation>&amp;Su</translation>
     </message>
     <message>
         <location filename="bootselectiondialog.ui" line="52"/>
         <source>U</source>
-        <translation type="unfinished"></translation>
+        <translation>U</translation>
```

Comment:
If you're leaving the shortcut-key as U, should the translation of `&amp;Up` be `S&amp;u` ? (IIRC the `&amp;` causes the next character to be underlined)